manim/mobject/geometry/labeled.py

```diff
@@ -160,3 +217,155 @@ def __init__(
         **kwargs: Any,
     ) -> None:
         super().__init__(*args, **kwargs)
+
+
+class LabeledPolygram(Polygram):
+    """Constructs a polygram containing a label box at its pole of inaccessibility.
+
+    Parameters
+    ----------
+    vertex_groups
+        Vertices passed to Polygram constructor.
+    label
+        Label that will be displayed on the Polygram.
+    precision
+        The precision used by the PolyLabel algorithm.
+    label_config
+        A dictionary containing the configuration for the label.
+        This is only applied if `label` is of type `str`.
```

Comment:
A text enclosed in single backticks is rendered in italic. In order to render it as code, enclose it in double backticks:
```suggestion
        This is only applied if ``label`` is of type ``str``.
```